src/ruby/yast/core_ext/ansi_string.rb

```diff
@@ -0,0 +1,26 @@
+module Yast
+  module CoreExt
+    module AnsiString
```

Comment:
looks good, just please add documentation for module and also examples of usage as it is first time we use refinements in yast.